Source/WebKit/UIProcess/API/Cocoa/WKWebViewTesting.mm

```diff
@@ -426,10 +426,10 @@ - (BOOL)_hasSleepDisabler
     return _page && _page->hasSleepDisabler();
 }
 
-- (NSString*)_scrollbarStateForScrollingNodeID:(uint64_t)scrollingNodeID isVertical:(bool)isVertical
+- (NSString*)_scrollbarStateForScrollingNodeID:(uint64_t)scrollingNodeID processID:(uint64_t)processID isVertical:(bool)isVertical
 {
     if (_page)
-        return _page->scrollbarStateForScrollingNodeID(scrollingNodeID, isVertical);
+        return _page->scrollbarStateForScrollingNodeID(scrollingNodeID, processID, isVertical);
```

Comment:
I think this is the point where we should re-assemble the WebCore::ScrollingNodeID and pass it to WebPageProxy:: scrollbarStateForScrollingNodeID as one parameter.

Reply:
Fixed.